src/server/pkg/testutil/context.go

```diff
@@ -0,0 +1,20 @@
+package testutil
+
+import (
+	"context"
+	"testing"
+)
+
+// NewTestContext returns a context scoped to the current test
```

Comment:
Is this really necessary? Go will just kill the binary if the timeout is exceeded. Also, if we keep this, the cleanup will need to be within the scope of the deadline check because it panics as is with a timeout of 0.

Reply:
removing this